dist/cornerstone.js

```diff
@@ -2213,13 +2219,10 @@ if(typeof cornerstone === 'undefined'){
 
     for(storedValue = image.minPixelValue; storedValue <= maxPixelValue; storedValue++)
     {
-      modalityLutValue = mlutfn(storedValue);
-      voiLutValue = vlutfn(modalityLutValue);
-      clampedValue = Math.min(Math.max(voiLutValue, 0), 255);
       if(!invert) {
-        lut[storedValue+ (-offset)] = Math.round(clampedValue);
+        lut[storedValue+ (-offset)] = vlutfn(mlutfn(storedValue));
       } else {
-        lut[storedValue + (-offset)] = Math.round(255 - clampedValue);
+        lut[storedValue + (-offset)] = vlutfn(mlutfn(storedValue));
```

Comment:
I think you are missing the (255 - x) here. Right now it appears both paths of the if statement are identical.